### ISSUE TITLE
Material You dynamic color doesn't work

### ISSUE THREAD

**Issue by user-naa** [bitwarden/android#3681]
### Bitwarden Beta

- [X] I'm using the new native Bitwarden Beta app and I'm aware that legacy .NET app bugs should be reported in [bitwarden/mobile](https://github.com/bitwarden/mobile)

### Steps To Reproduce

Open app on devices with Android 12+

### Expected Result

App is using Material You so accent color should match system

### Actual Result

The bottom menu color is lavender and the switches color is blue

### Screenshots or Videos

Bitwarden:
![Screenshot_20240806-105619_Bitwarden Beta](https://github.com/user-attachments/assets/cce4eeff-5144-47db-95c4-b6232223e37e)
Settings app (for example):
![Screenshot_20240806-123452_Settings](https://github.com/user-attachments/assets/48872159-1dc2-4543-bda5-f603d85074e0)

### Build Version

024.7.3 (18938)

### Environment Details

Android 14

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Krychaz** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by MillionsToOne** [comment]
Is it supposed to? This is more of a feature reques than a bug report because they don't support Material You.

**Comment by ThatOneCalculator** [comment]
I feel like it should, lol

**Comment by MillionsToOne** [comment]
Well, there is no Material You colors defined at least for what I can see. Maybe I'm blind I dunno. Regardless, there's already a feature request for Material You colors on their forums.

**Comment by LSeelig** [comment]
@MillionsToOne to confirm, you are saying that this isn't a bug, but simply unimplemented?

**Comment by MillionsToOne** [comment]
> @MillionsToOne to confirm, you are saying that this isn't a bug, but simply unimplemented?

Yes. Material Design 3 spec doesn't require Material You colors. Material Design 3 ≠ Material You ≠ Material Design 3. You can use brand colors with MD3 just what like Bitwarden and WhatsApp uses.
